packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -39,6 +39,66 @@ export function spawnAsync(
   });
 }
 
+export function execAsync(
+  command: string,
+  args: string[],
+  opts: SpawnOptions = {}
+) {
+  return new Promise<{ stdout: string; stderr: string; code: number }>(
+    (resolve, reject) => {
+      opts.stdio = 'pipe';
+
+      let stdout: Buffer = Buffer.from('');
+      let stderr: Buffer = Buffer.from('');
+
+      const child = spawn(command, args, opts);
+
+      child.stderr!.on('data', data => {
+        stderr = Buffer.concat([stderr, data]);
+      });
+
+      child.stdout!.on('data', data => {
+        stdout = Buffer.concat([stdout, data]);
+      });
+
+      child.on('error', reject);
+      child.on('close', (code, signal) => {
+        if (code !== 0) {
+          return reject(
+            new Error(
+              `Program "${command}" exited with non-zero exit code ${code} ${signal}.`
+            )
+          );
+        }
+
+        return resolve({
+          code,
+          stdout: stdout.toString(),
+          stderr: stderr.toString(),
+        });
+      });
+    }
+  );
+}
+
+export function spawnCommand(command: string, options: SpawnOptions = {}) {
+  if (process.platform === 'win32') {
+    return spawn('cmd.exe', ['/C', command], options);
+  }
+
+  return spawn('sh', ['-c', command], options);
+}
+
+export async function execCommand(command: string, options: SpawnOptions = {}) {
+  if (process.platform === 'win32') {
+    await spawnAsync('cmd.exe', ['/C', command], options);
+  } else {
+    await spawnAsync('sh', ['-c', command], options);
+  }
+
+  return true;
```

Comment:
Why does this always return true? I dont see any other return 🤔

Reply:
@styfle if it fails it'd throw, just like e.g. `runShellScript`